src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -161,7 +161,11 @@ export function updateLayeredMaterialNodeImagery(context, layer, node, parent) {
             // TODO: Handle error : result is undefined in provider. throw error
             const pitchs = extentsDestination.map((ext, i) => ext.offsetToParent(extentsSource[i], nodeLayer.offsetScales[i]));
             nodeLayer.setTextures(result, pitchs);
-            node.layerUpdateState[layer.id].success();
+            if (layer.source.isFileSource) {
```

Comment:
This change requires reflection, there may be a less specific solution.

Reply:
Yep, I'll close this PR that seems too quick and unthought